packages/client/src/assets/lemon-logo.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 40" class="lemonLogo" style="enable-background:new 0 0 60 40" xml:space="preserve">
```

Comment:
All of these should be in the public folder, but I'm not going to care too much about that right now since the goal is to 🚢

Reply:
Added a comment to Logo.js for revisit after MVP 🚢  :)